source/Calamari/Kubernetes/ResourceStatus/ResourceFinder.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Kubernetes.ResourceStatus.Resources;
+
+namespace Calamari.Kubernetes.ResourceStatus
+{
+    public class ResourceFinder
+    {
+        private readonly IVariables variables;
+        private readonly ICalamariFileSystem fileSystem;
+
+        public ResourceFinder(IVariables variables, ICalamariFileSystem fileSystem)
+        {
+            this.variables = variables;
+            this.fileSystem = fileSystem;
+        }
+
+        public IEnumerable<ResourceIdentifier> FindResources(string workingDirectory)
```

Comment:
This code is used only by the script wrapper to get resources from files so I've moved it to a separate class so we don't have to worry about it for the Raw Yaml Command.